docs/tutorial/intel-tcc/preparation.md

```diff
@@ -89,6 +89,11 @@ Make sure it says "Up" in the status column.
 docker compose ps
 ```
 
+```{note}
+After the Grafana container is up, it still takes a bit of time to create its internal database and completely start up.
+If the web interface is not immediately available, wait a minute and try again.
+```
+
 If everything is running, you can connect to Grafana by following these steps:
```

Comment:
````suggestion
```{note}
It may take a few minutes until Grafana starts up for the first time.
If the web interface is not immediately available, wait a minute and try again.
```

When everything is ready, you can access Grafana's web interface by following these steps:
````